clang/include/clang/CIR/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+set(MLIR_MAIN_SRC_DIR ${LLVM_MAIN_SRC_DIR}/../mlir/include ) # --src-root
+set(MLIR_INCLUDE_DIR ${LLVM_MAIN_SRC_DIR}/../mlir/include ) # --includedir
+set(MLIR_TABLEGEN_OUTPUT_DIR ${CMAKE_BINARY_DIR}/tools/mlir/include)
+include_directories(SYSTEM ${MLIR_INCLUDE_DIR})
+include_directories(SYSTEM ${MLIR_TABLEGEN_OUTPUT_DIR})
```

Comment:
Why do these need to be system include directories? Rather than adding these to global list of include paths, could we create an interface library and use it as an explicit dependency only for targets that need it?

Reply:
`SYSTEM` wasn't necessary! Removing in update.